stylesheets/github-light.css

```diff
@@ -37,7 +38,8 @@ SOFTWARE.
   color: #795da3;
 }
 
-.pl-smi /* variable.parameter.function, storage.modifier.package, storage.modifier.import, storage.type.java, variable.other */,
+.pl-smi /* variable.parameter.function, storage.modifier.package, storage.modifier.import,
+           storage.type.java, variable.other */,
```

Comment:
It's much easier to read if it's limited to 100 characters because you don't have to scroll over or (even worse) have word wrapping